src/core/annotation.js

```diff
@@ -1427,7 +1435,28 @@ class WidgetAnnotation extends Annotation {
 
     data.readOnly = this.hasFieldFlag(AnnotationFieldFlag.READONLY);
     data.required = this.hasFieldFlag(AnnotationFieldFlag.REQUIRED);
-    data.hidden = this._hasFlag(data.annotationFlags, AnnotationFlag.HIDDEN);
+
+    const print = this._hasFlag(data.annotationFlags, AnnotationFlag.PRINT);
+    const hidden = this._hasFlag(data.annotationFlags, AnnotationFlag.HIDDEN);
+    const noView = this._hasFlag(data.annotationFlags, AnnotationFlag.NOVIEW);
+
+    if (hidden) {
```

Comment:
Where do all these magical values, that are used below, come from?
My guess would be the scripting-API, hence a comment just above here explaining their origin would be a very good idea.